src/main/java/TestFile.java

```diff
@@ -32,4 +32,11 @@ private static void onMoreVariationOfIncorrectComparingToItself() {
             // whatever
         }
     }
+
+    private static void inappropriateIndentation() {
+        int x = 5;
```

Comment:
[Checkstyle] WARNING: 'member def type' have incorrect indentation level 8, expected level should be 4.